diff --git a/thirdparty/tesseract/CMakeLists.txt b/thirdparty/tesseract/CMakeLists.txt
--- a/thirdparty/tesseract/CMakeLists.txt
+++ b/thirdparty/tesseract/CMakeLists.txt
@@ -961,12 +961,7 @@ endif()
 
 if(ANDROID)
   add_definitions(-DANDROID)
-  find_package(CpuFeaturesNdkCompat REQUIRED)
-  target_include_directories(
-    libtesseract
-    PRIVATE "${CpuFeaturesNdkCompat_DIR}/../../../include/ndk_compat")
-  target_link_libraries(libtesseract PRIVATE CpuFeatures::ndk_compat)
 endif()
 
 # ##############################################################################
